src/App.tsx

```diff
@@ -1,14 +1,72 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo, useState,
+} from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { DropDownList } from './components/DropDownList';
+
+const debounce = (f: (value: string) => void, delay: number) => {
+  let timerId: NodeJS.Timeout;
```

Comment:
You can use a library like lodash for debounce function to ensure better performance and reliability.